src/battleground/ImageDrawer.js

```diff
@@ -13,11 +13,11 @@ class ImageDrawer {
 		this.ctx=ctx;
 	}
 
-	draw(i: Image, center: Vec, width: number, angle: number): void {
+	draw(i: Image, center: Vec, width: number, angle: number, alpha: number = 1.0): void {
```

Comment:
What is the purpose of the alpha?

Reply:
(I know you probably know already, but just in case): the alpha of an image is basically it's opaqueness. Something an alpha of 1.0 is 100% opaque and you can't see through it at all. Something with 0% alpha is completely transparent. 50% alpha (alpha = 0.5) is half-transparent.

This is really useful with smoke particles for example, so that they can fade out slowly to look like they are dispersing. If you watch the animated gif carefully you can see it.

Reply:
Yeah I didn't know that, thank you.